test/moves/gmax_chi_strike.test.ts

```diff
@@ -0,0 +1,123 @@
+import { Abilities } from "#enums/abilities";
+import { BattlerIndex } from "#enums/battler-index";
+import { Moves } from "#enums/moves";
+import { Species } from "#enums/species";
+import { GameManager } from "#test/testUtils/gameManager";
+import Phaser from "phaser";
+import { afterEach, beforeAll, beforeEach, describe, it, expect, vi } from "vitest";
+
+describe("Moves - G-Max Chi Strike grants a stackable crit boost", () => {
```

Comment:
NIT: This description is too long for the root `describe`:

```suggestion
describe("Moves - G-Max Chi Strike", () => {
```